app/scripts/services/oauth/types.ts

```diff
@@ -165,6 +182,8 @@ export type OAuthLoginResult = {
   socialLoginEmail: string;
   refreshToken: string;
   revokeToken: string;
+  accessToken: string;
+  metadataAccessToken: string;
 };
 
```

Comment:
nit: How would you feel about defining this as a derived type? This way we don't have to worry about code drift or having to track down runtime objects/methods that rely on this type if it ever needs to be updated.

```suggestion

export type OAuthRefreshToken = Pick<
  OAuthLoginResult,
  'idTokens' | 'accessToken' | 'metadataAccessToken'
>;
```

Reply:
Added a derived type here, https://github.com/MetaMask/metamask-extension/pull/34240/commits/8e65bddca5a0ef53db0a0fd8526a5dcb9ab6999b